prowler/providers/aws/services/codepipeline/codepipeline_project_repo_private/codepipeline_project_repo_private.py

```diff
@@ -0,0 +1,67 @@
+import ssl
+import urllib.error
+import urllib.request
+
+import boto3
+
+from prowler.lib.check.models import Check, Check_Report_AWS
+from prowler.providers.aws.services.codepipeline.codepipeline_client import (
+    codepipeline_client,
+)
+
+
+class codepipeline_project_repo_private(Check):
+    def execute(self):
+        findings = []
+        client = boto3.client("codestar-connections")
```

Comment:
You cannot use boto3 inside the checks, you will need to create another service class inside `codepipeline_service.py`. See `Logs` in `cloudwatch_service.py` as an example.